osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/infra/networking/ping/Pinger.java

```diff
@@ -81,4 +81,218 @@ public boolean ping(final String destination) {
     }
     return false;
   }
+
+  /*
+   * An external SIG analysis, recommended to make this not a public class.
+   * This class could be vulnerable to arbitrary command execution.
+   * The reason is that this class executes a command on the operating system.
+   */
+  class CommandExecutor {
```

Comment:
Deze zou nog private gemaakt kunnen worden.

Reply:
Makes the testability a lot harder, so i leave it package private